strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/logging/LoggingManagementController.java

```diff
@@ -162,20 +153,29 @@ public Object browseLogsDirectory(@PathVariable("path") Optional<String> path,
                 return getBadRequestResponseEntity("Requested path is not a directory!", acceptHeader);
             }
 
-            DirectoryListing directoryListing = getDirectoryListingService().fromPath(logsBaseDir, requestedLogPath);
+            UriComponentsBuilder baseBuilder = uriBuilder.getBuilder(ROOT_CONTEXT);
+
+            DirectoryListing directoryListing = directoryListingService.fromPath(baseBuilder.cloneBuilder().path("/browse"),
+                                                                                 baseBuilder.cloneBuilder().path("/download"),
+                                                                                 logsBaseDir,
+                                                                                 requestedLogPath);
 
             if (acceptHeader != null && acceptHeader.contains(MediaType.APPLICATION_JSON_VALUE))
             {
                 return ResponseEntity.ok(objectMapper.writer().writeValueAsString(directoryListing));
             }
 
-            String currentUrl = StringUtils.chomp(request.getRequestURI(), "/");
-            String downloadUrl = currentUrl.replaceFirst("/browse", "/download");
             boolean showBack = path.isPresent() && !StringUtils.isBlank(path.get());
+            String backPath = null;
+            if(showBack) {
```

Comment:
Cosmetic: Braces on new line, please.
```suggestion
            if(showBack)
             {
```